packages/block-editor/src/components/global-styles/image-settings-panel.js

```diff
@@ -67,6 +67,7 @@ export default function ImageSettingsPanel( {
 					panelId={ panelId }
 				>
 					<ToggleControl
+						__nextHasNoMarginBottom
```

Comment:
## Testing instructions 

In Global Styles, go to Blocks ▸ Image.

<img src="https://github.com/user-attachments/assets/ffacd383-ad5a-4ca3-b0f7-0f1d8ea53b8b" alt="Global Styles Image block" width="283">